### ISSUE TITLE
`build` generates precache entries for granular dependencies in the `bundled` build

### ISSUE THREAD

**Issue by kevinpschaaf** [Polymer/polymer-cli#116]
The precache list should only contain the shell and any bundled fragments (shards), none of the granular dependencies.

I'm seeing this for a simple app:

```
var PrecacheConfig = [
["/bower_components/app-layout/app-drawer-layout/app-drawer-layout.html","874364b05c4fd721e767594bbda00838"],
["/bower_components/app-layout/app-drawer/app-drawer.html","200c4c966babd7a35805ffbcac03327b"],
["/bower_components/app-layout/app-header-layout/app-header-layout.html","3dc2e1a80039ba2b37bda5117860672d"],
["/bower_components/app-layout/app-header/app-header.html","9b8a428a3f54b2148fd862fdc25f5231"],
["/bower_components/app-layout/app-scroll-effects/app-scroll-effects-behavior.html","5b12672a1f54257800b54401c783b8aa"],
["/bower_components/app-layout/app-scroll-effects/app-scroll-effects.html","334eac7f54a828baedbe8f09574571b7"],
["/bower_components/app-layout/app-scroll-effects/effects/blend-background.html","1df5c50b27bb943842d0743f37785de5"],
["/bower_components/app-layout/app-scroll-effects/effects/fade-background.html","4629b376fbc0e4de7953fe9614f453c5"],
["/bower_components/app-layout/app-scroll-effects/effects/material.html","79e57109d5b68cdef525ccfb8d95ddca"],
["/bower_components/app-layout/app-scroll-effects/effects/parallax-background.html","6fcade70bb0ea6e11f01ae528ac7d3c4"],
["/bower_components/app-layout/app-scroll-effects/effects/resize-snapped-title.html","7b740ace1c684bc69b94ac5bdcbc82f7"],
["/bower_components/app-layout/app-scroll-effects/effects/resize-title.html","840aee550e473a7395222cd3782521e7"],
["/bower_components/app-layout/app-scroll-effects/effects/waterfall.html","1c8fb19cef85687e5b4cbe0ec8dfbc4f"],
["/bower_components/app-layout/app-toolbar/app-toolbar.html","bca8ab0c780b2fdb89b40f89ec1803b5"],
["/bower_components/app-layout/helpers/helpers.html","a09e36bd26606edb78f1da95f50eb3df"],
["/bower_components/carbon-route/carbon-location.html","332fb8e4af06fc54368f0cbff9e5ebc6"],
["/bower_components/carbon-route/carbon-route-converter.html","9d5b76f5f0d3e625f86d5686d7c2d77d"],
["/bower_components/carbon-route/carbon-route.html","0fd0e57d3fa57f44928127e87882a81b"],
["/bower_components/iron-a11y-keys-behavior/iron-a11y-keys-behavior.html","5573ad30d50c5ae9eccd0f81fd4c0b68"],
["/bower_components/iron-behaviors/iron-button-state.html","ae3f511a1ca054ad4777a4a417c26a24"],
["/bower_components/iron-behaviors/iron-control-state.html","186df3effcc802cc6b64b59ed89d7008"],
["/bower_components/iron-flex-layout/iron-flex-layout.html","6653062aa51d14140fdcc5239b83dc76"],
["/bower_components/iron-icon/iron-icon.html","4238ef8af871c2872f7c1e7ffd57c66e"],
["/bower_components/iron-iconset-svg/iron-iconset-svg.html","b4e54eda060f011f4405d150e2e9c965"],
["/bower_components/iron-location/iron-location.html","b00f6a3f6ec4563ab5a0f530f6c7c3b3"],
["/bower_components/iron-media-query/iron-media-query.html","884063841b99c312597b4006782b33a4"],
["/bower_components/iron-meta/iron-meta.html","512213a62fc876291e12a232a5ce0578"],
["/bower_components/iron-pages/iron-pages.html","53556c62656b7b1a490a6c5fb50adb35"],
["/bower_components/iron-resizable-behavior/iron-resizable-behavior.html","701f1bc29d02a95abfc46bb67ceaf909"],
["/bower_components/iron-scroll-target-behavior/iron-scroll-target-behavior.html","2c88167f2df2e69d6c861334f16f425a"],
["/bower_components/iron-selector/iron-multi-selectable.html","a033160138980c462dde43fee8b65d9f"],
["/bower_components/iron-selector/iron-selectable.html","a766910a18f58367caeaca7098ff1483"],
["/bower_components/iron-selector/iron-selection.html","76f8a7b97df52ad5104c12c14ab77366"],
["/bower_components/iron-selector/iron-selector.html","e106c83c3473f4cd5528da00cc7c7572"],
["/bower_components/paper-behaviors/paper-inky-focus-behavior.html","04e1ab936983f71c3e4126213db444ba"],
["/bower_components/paper-behaviors/paper-ripple-behavior.html","54f9dd587dfc2db0be79121508d2c293"],
["/bower_components/paper-icon-button/paper-icon-button.html","7bc60cf7fd6a6f8e708e003a0b85528a"],
["/bower_components/paper-ripple/paper-ripple.html","de3113cc502b7254a4564564d4174c68"],
["/bower_components/paper-styles/color.html","4de8ef16b7ca20be8042469c80a9b20d"],
["/bower_components/paper-styles/default-theme.html","6ab9128559978d65df752ed494a099e9"],
["/bower_components/polymer/polymer-micro.html","bf388923ce6e851001f907013555724b"],
["/bower_components/polymer/polymer-mini.html","4b74358a6ae7db4d3e2f8edb14e5a891"],
["/bower_components/polymer/polymer.html","e162124b2426a07b694cb768c71a1302"],
["/bower_components/webcomponentsjs/webcomponents-lite.min.js","a1882f82ebfc212658591b4e817d8e03"],
["/index.html","146f5312a810829dfad86222786f0f82"],
["/manifest.json","708854a9ef654358facecadbf9ada8c4"],
["/shared-bundle.html","e89ee585fb0b2dd739975ffaf893b3d3"],
["/src/my-app.html","8b16bc3cec6d9d147f809ac4ca28592e"],
["/src/my-icons.html","91092e6abe361a1264d8972340124a3f"],
["/src/my-view1.html","874f6822e2cac1d92bf32f451104d191"],
["/src/my-view2.html","95450393613b51c7e2312c783a8054ee"],
["/src/my-view3.html","861b4fb5efdbf39d1320b3035a1a2db3"]];
```

**Comment by addyosmani** [comment]
Without taking a look at the source, it appears like we might be running precache against a full directory here (common with precache) vs being very explicit about what to cache - I'd suggest a static list of shell/other resources + minimal globs for everything else that's needed. You don't want to stuff too much into the Cache API :)

**Comment by justinfagnani** [comment]
@addyosmani We pass an explicit list built from static analysis. Here we're just including too many dependencies, when in the bundled version doesn't require them.

**Comment by justinfagnani** [comment]
Fixed by #100 which probably hadn't made it to the release this bug was reported against.
